algorithms/graph/dijkstra.py

```diff
@@ -34,3 +34,27 @@ def dijkstra(self, src):
                     dist[v] = dist[u] + self.graph[u][v]
 
         return dist
+
+    def dijkstraUsingHeap(self, src):
+        heap = BinaryHeap()
```

Comment:
Use snake case for functions.

Reply:
Om du inte gjort det, kör också `black` på filen så blir allt formatterat som resten.

Reply:
Fixed